service/controller/clusterapi/v29/cluster_resource_set.go

```diff
@@ -520,39 +520,6 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	// TODO use that code later when the new Node Pool ready status resource is
-	// used again.
```

Comment:
I think this is not valid anymore. Don't recall what we were thinking. Just dropping it now.

Reply:
Doesn't ring a bell here either. Go for it.